src/pages/ServiceDetails/ServiceMetrics.tsx

```diff
@@ -259,7 +268,8 @@ class ServiceMetrics extends React.Component<ServiceId, ServiceMetricsState> {
         }
       }
     };
-    return <LineChart id={id} title={{ text: title }} data={data} axis={axis} point={{ show: false }} />;
+    let obj = <LineChart id={id} title={{ text: title }} data={data} axis={axis} point={{ show: false }} />;
```

Comment:
Is it intentional change? (I'm asking, because I'd rather write the shorter 1-line version, but if there are reasons against it then ok)

Reply:
No sorry , I was playing with bind the c3 chart ^^